Makefile

```diff
@@ -257,6 +258,13 @@ $(EXT_AUTH)external_auth.pb.go $(EXT_AUTH)attribute_context.pb.go: $(EXT_AUTH)ex
 	              $(EXT_AUTH)*.proto \
 	              --gogofast_out=plugins=grpc,$(PROTOC_MAPPINGS):$(ENVOY_API)
 
+$(EXT_AUTH_V2_ALPHA)external_auth.pb.go: $(EXT_AUTH_V2_ALPHA)external_auth.proto $(EXT_AUTH)external_auth.proto
```

Comment:
Why would v2alpha .pb.go depend on v2 .proto?  IOW, is the `$(EXT_AUTH)external_auth.proto` dependency intended here?

Reply:
I had included this dependency because the v2alpha .proto imports the v2 .proto -- but looking at the generated go, this is just a standard go import, so nothing in the actual .go file depends on what's in v2. I'll drop the dependency in the Makefile.